wordpress-mobile/WordPress-iOS
# Integrated new analytics event for stats chart interaction

Fixes #11066.

To test:

- Checkout the branch and confirm that tests pass.
- Exercise the charts by selecting the Post Stats & Period charts, selecting different bars.
- In the console, you should see evidence that `stats_overview_bar_chart_tapped` is logged to the console. If a period is selected, it will appear in the properties with a `granularity` key. Otherwise, the properties dictionary will be empty.
- This should also be visible in _Tracks_ as `wpios_stats_overview_bar_chart_tapped`.

Update release notes:
